packages/wdio-local-runner/tests/localRunner.test.ts

```diff
@@ -414,7 +459,100 @@ test('should skip xvfb initialization when disabled in config', async () => {
         specs: ['/foo/bar.test.js'],
         execArgv: [],
         retries: 0,
-    })
+    } as any)
 
     expect(mockInit).not.toHaveBeenCalled()
 })
+
+test('should pass xvfbAutoInstall:true to XvfbManager', async () => {
+    const xvfb = await import('@wdio/xvfb')
+    const runner = new LocalRunner(
+        undefined as never,
```

Comment:
Why is this set as never?

Reply:
The first param to the LocalRunner constructor is not used, I guess we could remove it.

Reply:
Hmmmm https://github.com/webdriverio/webdriverio/actions/runs/17142093207/job/48631319971?pr=14700

Reply:
BrowserRunner extends LocalRunner and uses those options.